src/components/flow/exit/Exit.tsx

```diff
@@ -252,7 +253,19 @@ export class ExitComp extends React.PureComponent<ExitProps, ExitState> {
     }
   }
 
-  public getName(): { name: string; localized?: boolean } {
+  public getName(): { name: string; localized?: boolean; disabled?: boolean } {
+    let disabled = false;
+    if (
+      this.props.node.actions[0] &&
+      this.props.node.actions[0].type &&
+      this.props.node.actions[0].type === Types.enter_flow &&
+      this.props.categories.some(
```

Comment:
If the expired property is unique to enter a flow than the first three checks can be removed